src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java

```diff
@@ -31,13 +36,13 @@ public class InputFieldAnnotations {
 
   @SomeAnnotation1 @SomeAnnotation2 boolean test = false;
 
-  @SuppressWarnings("all") @SomeAnnotation3(x = 0) float pi = 3.14f;
+  @Deprecated @SomeAnnotation3(x = 0) float pi = 3.14f;
 
   @SomeAnnotation1 @SomeAnnotation3(x = 14)
   /** testing. */ // violation 'Javadoc comment is placed in the wrong location.'
   List<String> list = new ArrayList<>();
 
-  @SuppressWarnings("all")
+  @Deprecated
```

Comment:
changed annotation because `SuppressWarnings` was suppressing the violation

Reply:
Please give more details. I'm trying to understand why the presence of `SuppressWarnings` affects the violation.

Reply:
uh, such a nuance.
https://checkstyle.org/checks/annotation/suppresswarningsholder.html#Description

>. It is possible to suppress all the checkstyle warnings with the argument "all".

please make annotation like  `@SuppressWarnings("bla")`

Reply:
oh, we also have documentation for this. I didn't knew about it, thanks for sharing the link.

I have changed the annotation to `@SuppressWarnings("bla")`